api-docs/rst/dev-guide/concepts.rst

```diff
@@ -683,14 +796,14 @@ Schedule-based configurations
 You can configure Autoscale to be triggered based on a user-defined
 schedule that is specified in one or more policies.
 
-This configuration option is helpful if you know that your Cloud Servers
-deployment will need additional resources during certain peak times. For
-example, if you need additional server resources during the weekend, you
-can define a policy that adds 50 servers on Friday evening and then
-removes these servers again on Sunday evening to return to a regular
+This configuration option is helpful if you know that your scaling group
+will need additional resources during certain peak times. For
+example, if you need additional resources during the weekend, you
+can define a policy that adds 50 units on Friday evening and then
+removes these units again on Sunday evening to return to a regular
 operational state.
 
```

Comment:
if you know that our scaling group requires additional resources during certain peak times. For example, if you need additional resources during the weekend, you can define a policy to add 50 units on Friday evening and remove those units on Sunday evening to return to the operational state for weekdays.